cardano-cli/src/Cardano/CLI/Shelley/Orphans.hs

```diff
@@ -62,6 +63,18 @@ instance Crypto c => ToJSON (TxOut c) where
       , "amount" .= amount
       ]
 
+-- duplicated in cardano-config
```

Comment:
What does it mean to be duplicated in `cardano-config`.  Does this code appear there also and that we are intending for this duplication to continue going forward?

Reply:
Updated the comment to be more obvious.